skeleton-tests/test_addon_theme.py

```diff
@@ -30,7 +30,7 @@ def test_addon_theme(tmpdir, capsys, config):
 
     # generate template addon:
     config.template = 'addon'
-    config.package_name = 'plonetheme.task'
```

Comment:
I think we should stay with plonetheme.* here, because thats what most people we use.
I there are any isort issue because of the the name, i would prefer to resort them for the theme templates, if doable.

Reply:
@MrTango I will check.

Reply:
?

Reply:
@MrTango I was a bit caught up. Let me dive into it.

Reply:
I updated the name. Let me know how that works for you.